var/spack/repos/builtin/packages/crtm/package.py

```diff
@@ -58,3 +64,17 @@ class Crtm(CMakePackage):
     # Branch release/crtm_jedi_v2.4.0
     version("v2.4-jedi.1", commit="82223419fdb479d76c2f2109c2b704e1d9618f22")
     version("v2.4-jedi.2", commit="62831cbb6c1ffcbb219eeec60e1b1c422526f597")
+
+    def url_for_version(self, version):
+        if self.spec.satisfies("@v3") or version >= Version("3.0.0"):
+            return "https://github.com/JCSDA/crtmv3/archive/refs/tags/{}.tar.gz".format(version)
+        else:
+            return "https://github.com/JCSDA/crtm/archive/refs/tags/{}.tar.gz".format(version)
```

Comment:
FYI. We do encourage use of Python f-strings (https://docs.python.org/3/tutorial/inputoutput.html) when feasible, which we're moving to in the core.